assembl/static2/js/app/constants.js

```diff
@@ -180,18 +187,14 @@ export const ADMIN_MENU = {
     title: 'administration.landingpage',
     sectionId: '1',
     subMenu: {
-      // manageModules: {
-      //   title: 'administration.landingPage.manageModules.title',
-      //   sectionId: '3'
-      // },
       header: {
         title: 'administration.landingPage.header.title',
         sectionId: '1'
-      },
-      timeline: {
-        title: 'administration.landingPage.timeline.title',
-        sectionId: '2'
       }
+      // manageModules: {
```

Comment:
Is it useful to keep this commented code ?

Reply:
Yes, at the time we decided that we would keep the code until it will be use for the landing page